### DISCUSSION TITLE
New beta version 1.0.8-beta10

### DISCUSSION THREAD

**Discussion by pierreguillot** [pierreguillot/Camomile#284] [Announcement]
Hi,

I published a new temporary version ~1.0.8-beta9~ [1.0.8-beta10](https://github.com/pierreguillot/Camomile/releases/tag/v1.0.8-beta10). This beta version should be stable but again, I didn't manage to do all the features and fixes that I wanted to do for the version [1.0.8](https://github.com/pierreguillot/Camomile/projects/12) yet.

Your feedback would be much appreciated!

- Updates Pd to 0.52-0
- Updates Juce version (6.1.2)
- Adds Apple M1 support 
- https://github.com/pierreguillot/Camomile/issues/280
- https://github.com/pierreguillot/Camomile/issues/197
- https://github.com/pierreguillot/Camomile/issues/201
- https://github.com/pierreguillot/Camomile/issues/203
- https://github.com/pierreguillot/Camomile/issues/199
- https://github.com/pierreguillot/Camomile/issues/210
- https://github.com/pierreguillot/Camomile/issues/207
- https://github.com/pierreguillot/Camomile/issues/196
- https://github.com/pierreguillot/Camomile/issues/208
- https://github.com/pierreguillot/Camomile/issues/218
- https://github.com/pierreguillot/Camomile/issues/222
- https://github.com/pierreguillot/Camomile/issues/229

**Comment by porres** [comment]
Hi, really looking forward to this, any idea when we should have a final version compatible to Pd 0.52?
